com.reprezen.swagedit.openapi3/resources/templates.xml

```diff
@@ -0,0 +1,328 @@
+<templates>
+	<template name="OpenApi version" 
+       id="com.reprezen.swagedit.openapi3.templates.openapi_template" 
+       description="OpenAPI version declaration" 
+       context="com.reprezen.swagedit.openapi3.templates.root" 
+       enabled="true">openapi: "3.0"</template>
+
+<!--
+	INFO  
+ -->
+
+	<template name="OpenApi info" 
+       id="com.reprezen.swagedit.openapi3.templates.info_template" 
+       description="Provides descriptive metadata about this API specification" 
+       context="com.reprezen.swagedit.openapi3.templates.root" 
+       enabled="true">info:
+  version: 1.0
+  title: ${API_title}
+  description: ${API_description}
+  termsOfService: ${terms}
+  contact:
+    name: ${contact_name}
+    url: ${contact_url}
+    email: ${contact_email}
+  license:
+    name: ${MIT}
+    url: http://opensource.org/licenses/MIT</template>
+
+	<template name="OpenApi contact" 
+       id="com.reprezen.swagedit.openapi3.templates.info_contact_template" 
+       description="Provides contact metadata about this API specification" 
+       context="com.reprezen.swagedit.openapi3.templates.info.contact" 
+       enabled="true">name: ${contact_name}
+    url: ${contact_url}
+    email: ${contact_email}</template>
+
+<!--
+	SERVERS 
+ -->
+
+	<template name="OpenApi servers" 
+       id="com.reprezen.swagedit.openapi3.templates.servers_template" 
+       description="OpenAPI servers declaration" 
+       context="com.reprezen.swagedit.openapi3.templates.root" 
+       enabled="true">servers:
+  - uri: ${uri}
+    description: ${description}</template>
+ 
+ <!--
+ 	PATHS 
+  -->
+
+     <template name="Object resource" 
+       id="com.reprezen.swagedit.openapi3.templates.collection_resource_template" 
+       description="object resource template" 
+       context="com.reprezen.swagedit.openapi3.templates.paths" 
+       enabled="true">/${resource}:
+  get:
+    responses:
+      '${200}':
+        description: OK
+        schema:
+          $$ref: ${model}</template>
+
+<!--
+	PATH ITEMS 
+ -->
+ 
+	<template name="get" 
+       id="com.reprezen.swagedit.openapi3.templates.path_get_template" 
+       description="GET template" 
+       context="com.reprezen.swagedit.openapi3.templates.path_item" 
+       enabled="true">get:
+      summary: Read
+      description: Provide details for the entire list (for collection resources) or an item (for object resources)
+      responses:
+        '${200}':
+          description: ${OK}
+          schema:
+            $$ref: ${model}</template>
+
+     <template name="post" 
+       id="com.reprezen.swagedit.openapi3.templates.path_post_template" 
+       description="POST template" 
+       context="com.reprezen.swagedit.openapi3.templates.path_item" 
+       enabled="true">post:
+      summary: Create
+      description: Create a new item
+      parameters:
+        - name: ${request}
+          in: body
```

Comment:
`body` parameters are [not allowed in v3](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/3.0.md#parameter-locations), they are replaced by [requestBody](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/3.0.md#requestBodyObject)